src/loans/liquidable_debt.py

```diff
@@ -439,166 +437,51 @@ def process_kamino_loan_states(loan_states: list[KaminoLoanStates]) -> pandas.Da
-        list(set_of_debt_tokens)
-    )
-    debt_token_prices.update({'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v': 1})
-    for supply_data in debt_complete_data:
-        # price
-        if not supply_data['token']:
-            supply_data['price'] = None
-        else:
-            supply_data['price'] = debt_token_prices[supply_data['token']]
-        # decimals
-        supply_data['decimals'] = src.loans.helpers.get_decimals(supply_data['token'])
-
-    for supply_data in collateral_complete_data:
-        loan_states[f"collateral_{supply_data['supply_account']}"] = loan_states['collateral'].apply(
-            lambda x: x[supply_data['supply_account']] if supply_data['supply_account'] in x else decimal.Decimal('0'))
-
-        ltv = supply_data['config']['reserve']['config']['loanToValueRatio']
-        price = supply_data['price']
-        decimals = supply_data['decimals']
-
-        if price:
-            loan_states[f"collateral_usd_{supply_data['supply_account']}"] = (
-                    loan_states[f"collateral_{supply_data['supply_account']}"].astype(float)
-                    / (10 ** decimals)
-                    * (ltv / 100)
-                    * price
-            )
-
-    for supply_data in debt_complete_data:
-        loan_states[f"debt_{supply_data['supply_account']}"] = loan_states['debt'].apply(
-            lambda x: x[supply_data['supply_account']] if supply_data['supply_account'] in x else decimal.Decimal('0'))
-
-        price = supply_data['price']
-        decimals = supply_data['decimals']
-        if price:
-            loan_states[f"debt_usd_{supply_data['supply_account']}"] = (
-                    loan_states[f"debt_{supply_data['supply_account']}"].astype(float)
-                    / (10 ** decimals)
-                    * price
-            )
-
-    COLLATERAL_TOKENS = {i['underlying_token'] for i in collateral_complete_data if i['underlying_token']}
-    DEBT_TOKENS = {i['token'] for i in debt_complete_data}
-    # COLLATERAL_TOKENS = [
-    #     'So11111111111111111111111111111111111111112',
-    #     'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v',
-    #     'J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn',
-    #     'bSo13r4TkiE4KumL71LsHTPpL2euBYLFx6h9HP3piy1',
-    #     '27G8MtK7VtTcCHkpASjSDdkWWYfoqT6ggEuKidVJidD4',
-    # ]
-    # DEBT_TOKENS = [
-    #     'So11111111111111111111111111111111111111112',
-    #     'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v',
-    #     'bSo13r4TkiE4KumL71LsHTPpL2euBYLFx6h9HP3piy1,',
-    #     'Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB',
-    #     'J1toso1uCk3RLmjorhTtrVwY9HJ7X8V9yYac6Y7kGCPn',
-    # ]
-
-    all_data = pandas.DataFrame()
-    for collateral_token, debt_token in itertools.product(COLLATERAL_TOKENS, DEBT_TOKENS):
-        logging.info(f"processing liquidable debt for {collateral_token} - {debt_token}")
+    # get all present collateral/debt token pairs
+    relevant_pairs = state.find_relevant_mint_collateral_pairs()
```

Comment:
The method seems to have been renamed.

Reply:
thanks, i forgot to add this module to my last commit.